dev-support/bin/tez_run_example.sh

```diff
@@ -0,0 +1,119 @@
+
+# This script is used to set up a local Hadoop and Tez environment for running a simple word count example.
+# Prerequisites
+# 1. java is installed and JAVA_HOME is set
+# 2. ssh localhost works without password
+
+# configure this if needed, by default it will use the latest stable versions in the current directory
+export TEZ_VERSION=$(curl -s "https://downloads.apache.org/tez/" | grep -oP '\K[0-9]+\.[0-9]+\.[0-9]+(?=/)' | sort -V | tail -1) # e.g. 0.10.4
+export HADOOP_VERSION=$(curl -s "https://downloads.apache.org/hadoop/common/" | grep -oP 'hadoop-\K[0-9]+\.[0-9]+\.[0-9]+(?=/)' | sort -V | tail -1) # e.g. 3.4.1
```

Comment:
shouldn't the hadoop version should be from the pom? not always the latest version is gonna work with Tez

Reply:
good question, depends on what we want to achieve with this script, here is what I can think of:
1. get hadoop from the tez pom.xml as you adviced
2. both HADOOP_VERSION and TEZ_VERSION could be used from env if already defined (making the user able to define any for random experience)

Reply:
I am like if the user defines it use it else get it from the POM, I believe that is what the Hive docker build script does that as well.

Reply:
yeah, makes sense, let me do the same